infra/docker-evm-blockhain/start.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# Set log file
+LOG_FILE="debug.log"
+
+# Clear log file content if it exists
+> $LOG_FILE
+
+# Set network ID, block time, and ports
+NETWORK_ID=14578
+BLOCK_TIME=2
+
+BOOTNODE_PORT=30301
+NODE1_PORT=30310
+NODE1_HTTP_PORT=8040
+NODE1_AUTH_PORT=8551
+NODE2_PORT=30311
+NODE2_HTTP_PORT=8041
+NODE2_AUTH_PORT=8552
+
+# Create directories for the network, nodes, and bootnode
+mkdir -p bootnode
+mkdir -p node1
+mkdir -p node2
+
+# Generate genesis.json file
+cat << EOF > genesis.json
```

Comment:
Why we are creating an empty genesis.json and then add data using cat? Shouldn't we create the genesis.json in the repository with this content? is it due to NETWORK_ID, BLOCK_TIME? If so, why do we need empty file? Can't we create the file from the script itself? 

the problem is, when someone takes a checkout and run this scripts, there are files modified in the checkout, which might get committed back to repository accidentally. We should avoid the files in repository getting modified by these scripts.

Reply:
I removed the genesis.json file from the directory since we are already creating the `genesis.json` file inside the Docker container using the `start.sh` script.